app/src/main/res/layout/fragment_login.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:gravity="center"
+        android:orientation="vertical"
+        android:paddingVertical="@dimen/static_spacing_large_extra">
+
+
+        <ImageView
+            android:id="@+id/logoImage"
+            android:layout_width="fill_parent"
+            android:layout_height="100dp"
```

Comment:
Put this value inside the dimen file 👍

Reply:
Yep! About these: You'll find some files like `dimens.xml`, `strings.xml`, etc. You can put strings and dimensions in those files to make them reusable and avoid hardcoding them.